catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/bower.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "registry-admin-remote-ui",
+  "version": "0.0.1",
+  "private": true,
+  "dependencies": {
+    "backbone-relational": "0.8.8",
```

Comment:
If you're using backbone associations, which it appears you are, then you don't need relational.